main.go

```diff
@@ -471,6 +302,11 @@ func mainErr(args []string) error {
 		return fmt.Errorf("unknown command: %q", args[0])
 	}
 
+	if err := loadShared(); err != nil {
+		return err
+	}
+	opts = getOptions()
```

Comment:
similar to my other comment, loadShared could do this for you. Or you could just do `opts = &cache.Options` here. I think `getOptions` is a bit redundant as a standalone func